src/api/config.js

```diff
@@ -1,31 +1,53 @@
 'use strict'
 
-const argCommand = require('../cmd-helpers').argCommand
-
 module.exports = (send) => {
   return {
-    get: argCommand(send, 'config'),
-    set (key, value, opts, cb) {
-      if (typeof (opts) === 'function') {
-        cb = opts
+    get (key, callback) {
+      if (typeof key === 'function') {
+        callback = key
+        key = undefined
+      }
+
+      if (!key) {
+        return send('config/show', null, null, null, true, callback)
```

Comment:
It is happening :) https://github.com/ipfs/js-ipfs-api/pull/313/commits/43a1dfab4322dd06af7fd158dae29e7da71d4d86